codewof/programming/content/en/playing-card/question.md

```diff
@@ -0,0 +1,11 @@
+# Playing card
+
+Write a program that prompts the user for a card.
+If the user enters a type of playing card, your program should **print** `That's a playing card!`.
+
+Your program should keep prompting for a card until the user enters something that is not a type of playing card.
+Then your program should **print** `That's not a playing card!` and quit.
+
+For this question, a type of playing card is any integer between 2 and 10 (inclusive), or a string `A`, `J`, `Q` or `K`.
+
+*Hint: It might be useful to use `while {variable} in {list}:` for your comparison.*
```

Comment:
Put an asterisk right after the word *use* and right before the word *for* so that `while {variable} in {list}:` is formatted as code (unless it already is? I can't test it myself right now)

Reply:
![image](https://user-images.githubusercontent.com/25916475/65200847-888b4680-dadc-11e9-9796-35963f53b9dd.png)

It does work, but should I change it anyway?

Reply:
Nope, don't worry about it - I just couldn't remember if it worked or not :)